src/handlers/ar.rs

```diff
@@ -99,11 +122,22 @@ impl super::Processor for Ar {
                 let mode = std::str::from_utf8(&buf[40..48])?.trim_end_matches(' ');
                 let mode = mode.parse::<u64>()?;
 
-                debug!("{}: file {:?}, mtime={}, {}:{}, mode={:o}, size={}",
-                       io.input_path.display(), name, mtime, uid, gid, mode, size);
-
-                if self.config.source_date_epoch.is_some() && mtime > self.config.source_date_epoch.unwrap() {
-                    let source_date_epoch_str = format!("{:<12}", self.config.source_date_epoch.unwrap());
+                debug!(
+                    "{}: file {:?}, mtime={}, {}:{}, mode={:o}, size={}",
+                    io.input_path.display(),
+                    name,
+                    mtime,
+                    uid,
+                    gid,
+                    mode,
+                    size
+                );
```

Comment:
Not better.

Reply:
Yes, I agree.  Though the real fix here is https://github.com/keszybz/add-determinism/pull/15 I think.